docs/docusaurus.config.js

```diff
@@ -136,37 +147,49 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      colorMode: {
+        defaultMode: "light",
+        disableSwitch: true,
```

Comment:
why?

Reply:
New design doesn't have light/dark colour modes, dev did enquire if this has been approved to removed and told that it was.

Reply:
Can't recall that approval, but fine by me - hereby approved. We can re-add it later / enable it only for the docs pages if we want.